sbt/src/main/scala/stryker4s/sbt/SbtMutantRunner.scala

```diff
@@ -0,0 +1,73 @@
+package stryker4s.sbt
+import java.nio.file.Path
+
+import better.files.File
+import sbt.Keys._
+import sbt._
+import stryker4s.config.Config
+import stryker4s.model._
+import stryker4s.run.MutantRunner
+import stryker4s.run.process.ProcessRunner
+
+class SbtMutantRunner(state: State, processRunner: ProcessRunner)(implicit config: Config)
+    extends MutantRunner(processRunner) {
+
+  val extracted: Extracted = Project.extract(state)
+
+  override def runInitialTest(workingDir: File): Boolean = {
+    val newState = extracted.appendWithoutSession(settings(workingDir), state)
+    Project.runTask(test in Test, newState) match {
+      case None =>
+        throw new RuntimeException(
```

Comment:
Shouldn't the initialTestRunException be used here?

Reply:
[According to sbt](https://www.scala-sbt.org/1.x/docs/Build-State.html#Running+tasks), a None indicates that the taskKey is invalid. Which I guess means sbt doesn't recognize the `test` command. 

Which is a lot bigger problem than 'your tests don't succeed', which is what `InitialTestRunException` is for. :p

Reply:
Fixed